Cannot Import Types

When I import something from primitive-geometry like so:
```typescript
import { cube } from 'primitive-geometry';
```
I get the following error in my editor and when running `yarn tsc`:

> error TS7016: Could not find a declaration file for module 'primitive-geometry'. '/path/to/project/node_modules/primitive-geometry/index.js' implicitly has an 'any' type.
  There are types at '/path/to/project/node_modules/primitive-geometry/types/index.d.ts', but this result could not be resolved when respecting package.json "exports". The 'primitive-geometry' library may need to update its package.json or typings.

The "Are The Types Wrong" checking tool also reports that [the types for this project are not specified correctly](https://arethetypeswrong.github.io/?p=primitive-geometry%402.10.0):
> ❌ [No types](https://github.com/arethetypeswrong/arethetypeswrong.github.io/blob/main/docs/problems/UntypedResolution.md)
>   Import resolved to JavaScript files, but no type declarations were found.
> 
> ⚠️ [ESM (dynamic import only)](https://github.com/arethetypeswrong/arethetypeswrong.github.io/blob/main/docs/problems/CJSResolvesToESM.md)
>   A require call resolved to an ESM JavaScript file, which is an error in Node and some bundlers. CommonJS consumers will need to use a dynamic import.

<details>
<summary>My tsconfig.json</summary>
<pre>
{
  "compilerOptions": {
    "target": "ES2020",
    "useDefineForClassFields": true,
    "lib": ["ES2020", "DOM", "DOM.Iterable"],
    "module": "ESNext",
    "skipLibCheck": true,

    "types": ["@webgpu/types"],

    /* Bundler mode */
    "moduleResolution": "bundler",
    "allowImportingTsExtensions": true,
    "resolveJsonModule": true,
    "isolatedModules": true,
    "noEmit": true,
    "jsx": "react-jsx",

    /* Linting */
    "strict": true,
    "noUnusedLocals": true,
    "noUnusedParameters": true,
    "noFallthroughCasesInSwitch": true
  },
  "include": ["src"],
  "references": [{ "path": "./tsconfig.node.json" }]
}
</pre>
</details>

Adding `"primitive-geometry"` or `"primitive-geometry/types"` to `types` in `tsconfig.json` does not clear this error.

As a workaround, I can ignore the error and forgo the type safety.

--- a/package.json
+++ b/package.json
@@ -51,7 +51,12 @@
   "license": "MIT",
   "author": "Damien Seguin (https://github.com/dmnsgn)",
   "type": "module",
-  "exports": "./index.js",
+  "exports": {
+    ".": {
+      "types": "./types/index.d.ts",
+      "default": "./index.js"
+    }
+  },
   "main": "index.js",
   "types": "types/index.d.ts",
   "scripts": {

